_includes/footer.html

```diff
@@ -19,7 +19,8 @@
       </div>
       <!-- copyright info -->
       <div class="twelve columns site-footer-cc">
-        <p> &copy; {{ 'now' | date: "%Y" }}, Toronto Mesh. Our <a href="/code-of-conduct/">Code of Conduct</a>.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
+        <p> &copy; {{ 'now' | date: "%Y" }}, Toronto Mesh.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
+        <p><a href="/code-of-conduct/">Code of Conduct</a></p>
```

Comment:
add "." as end.

Also, could we keep these elements closer together? 
I'm not tied to the ordering. But I think having "Code of Conduct? as a dangling paragraph feels unbalanced.

Reply:
how bout removing below  since its in submenu now?

Reply:
(unless we want to add like a privacy policy or terms or something then we can list them all there too)

Reply:
Code of Conduct feels a bit off by itself. Maybe we can incorporate into a sentence?